src/server/routes/authRoutes.js

```diff
@@ -12,4 +12,9 @@ authRouter.post(
   AuthController.loginUser()
 );
 
+authRouter.post(
+  '/api/user/password/forgot',
```

Comment:
@ayodelevm would this make more sense if it were
`/api/user/account/recover`

Reply:
Ok. but what of `api/user/password/recover` since for the reset password is `api/user/password/reset`?